CatchMe/CatchMe/Source/Cells/Report/CatchuCVC.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CatchuCVC.swift
+//  CatchMe
+//
+//  Created by SHIN YOON AH on 2021/07/06.
+//
+
+import UIKit
+
+import SnapKit
+
+class CatchuCVC: UICollectionViewCell {
+    static let identifier = "CatchuCVC"
+    
+    // MARK: - Properties
+    var characterImage = UIImageView()
+    
+    // MARK: - Life Cycle
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configUI()
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Custom Method
+    private func configUI() {
+        characterImage.backgroundColor = .systemGray
+    }
+    
+    private func setupLayout() {
+        addSubview(characterImage)
+        
+        characterImage.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.centerY.equalToSuperview()
+            make.height.equalTo(150)
+            make.width.equalTo(150)
```

Comment:
선배야.. 이거 조심스럽게 제안해볼게.. 

```suggestion
            make.centerX.centerY.equalToSuperview()
            make.width.height.equalTo(150)
            
```

Reply:
예리하네.. 코드 좀 보나봐?